### ISSUE TITLE
A11y_Model Builder_Image Classification_data_Usability: Narrator/NVDA not announcing the "supported file formats" information to the select a folder text field.

### ISSUE THREAD

**Issue by Harika1702** [dotnet/machinelearning-modelbuilder#2670]
### GitHub Tags
#A11ySev4; #A11yTCS; #A11yUsable; #DesktopApp; #Win11; #Visual Studio Client; #SH_ModelBuilder_Win32_May2023; #FTP; #Win32; #Benchmark; #Screen Reader; #Narrator; #NVDA;

### Environment
Application Name: Model Builder
Microsoft Visual Studio Enterprise 2022 (64-bit) - Current
Version 17.6.1
Microsoft Windows 11 Enterprise Build 22624.1755
Narrator/NVDA Version 2023.0.1

### Repro Steps

1. Launch the Visual Studio with at least one of the .NET workload.
2. Tab till enable ML.NET Model Builder (Tools -> Options -> Environment -> Preview Features) and select it.
3. Create a new .NET project or open an existing .NET project.
4. Open context menu (Right-click) of the project in the solution explorer.
5. Tab till Add and then tab till Machine Learning Model and press enter.
6. Model Builder dialog will open. Tab till Scenario tab.
7. Select Scenario screen will appear.
8. Tab till "Image Classification" and press enter.
9. Setup environment screen will appear, Tab till Local CPU tile and select it.
10. Tab till Next button and press enter.
11. Data screen will appear.
12. Tab till "Select a folder" text field.
13. Now Turn on Narrator/NVDA.
14. Now verify that Narrator/NVDA not announcing the "supported file formats" information to the select a folder text field.

### Actual Result
Narrator/NVDA not announcing the "supported file formats" information to the select a folder text field.
**Note: Issue is also observed with NVDA.**

**Note: We are facing similar issue throughout the application**

### Expected Result
Narrator/NVDA should announce the "supported file formats" information to the select a folder text field.

### User Impact
Screen reader users will not get to know and gets confused which format that folder will take for the select a folder field.

### Attachment
![Support format_Narrator](https://github.com/dotnet/machinelearning-modelbuilder/assets/100270074/51a07e8f-9069-406b-b44f-056dd1ae4d46)
NVDA
![Support format_NVDA](https://github.com/dotnet/machinelearning-modelbuilder/assets/100270074/824e0d0e-d0cc-4df1-b775-fdb53448f32a)
Code snippet:
<img width="960" alt="Bug_Support format_Code snippet" src="https://github.com/dotnet/machinelearning-modelbuilder/assets/100270074/81e32298-01d4-4ba7-a891-44ea25f979f7">

**Comment by zewditu** [comment]
@Harika1702 , ready to validate by Model Builder version 16.17.0.2332602

**Comment by Yogi2204** [comment]
#Regressed:06-29-23; #Closed;
Verified the issue in below environment:
Visual Studio : Microsoft Visual Studio Enterprise 2022 (64-bit) - Current Version 17.6.3
Model Builder : Version 17.17.0.2332602

The issue is fixed, hence closing the bug. Please find attachment below for reference. Thanks!
[Recording-20230629_172617.webm](https://github.com/dotnet/machinelearning-modelbuilder/assets/130456851/6f2881cd-3f04-4405-b17c-8bf49e874189)
[Recording-20230629_174131.webm](https://github.com/dotnet/machinelearning-modelbuilder/assets/130456851/b517a412-a4a6-4bbe-acb1-92d666ab2f6e)
